<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>previewpage</class>
 <widget class="QMainWindow" name="previewpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#previewpage{
background-color: rgb(159,197,232);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color: rgb(159,197,232);
}</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>791</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_subject">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>主   题：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recever">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>收件人：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_subject">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>701</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_subject{
border-radius:6px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_recever">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_recever{
border-radius:6px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_subject</tabstop>
  <tabstop>lineEdit_recever</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
